concent-builder/templates/jenkins/Jenkinsfile.j2

```diff
@@ -7,96 +7,99 @@ properties([
     pipelineTriggers([githubPush()])
 ])
 
-// This is a process of fetching and checkouting concent-deployment repository
-// before running playbooks in properly versions.
-stage ('Checkout repository') {
-    dir('/home/jenkins/concent-deployment/') {
-        git url:
-        '{{ jenkins_github_url_address }}',
-        branch: 'ci'
+catchError {
```

Comment:
Won't this silence errors?

Reply:
This is the same structure like `try.. catch.. finally`. This helps to cleanup workspace if the error occurs and it's not silence it.